.github/workflows/tests-multi-user.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Multi User Tests
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
+  workflow_dispatch:
+jobs: 
+  test-multiple-aids:
+    name: Run multi user integration test using keria:${{ matrix.keria-version }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: signify-ts-test/
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        keria-version: ["0.2.0-dev3"]
+        node-version: ["20"]
+    env:
+      KERIA_IMAGE_TAG: ${{ matrix.keria-version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}       
+      - name: install deps
+        run: |
+          npm cache clean --force
+          npm set registry https://registry.npmjs.org/
+          npm ci    
+      - name: Build
+        run: npm run build      
+      - name: Start dependencies
+        run: |
+          docker compose down
+          docker compose up verify -d --pull always
+      - name: Run the vlei issuance test
+        run: |
+          LEI="875500ELOZEL05BVXV37" SIGNIFY_SECRETS="D_PbQb01zuzQgK-kDWjqy,BTaqgh1eeOjXO5iQJp6mb,nuuUBIFb9bfUB98bfkj8f,nf98hUHUy8Vt5tvdyaYV7" TEST_ENVIRONMENT="docker" npx jest ./singlesig-vlei-issuance.test.ts
```

Comment:
this yml should reference your script so we only maintain this test case in one place, that can be run manually.

Reply:
Done